pywt/multidim.py

```diff
@@ -19,7 +19,7 @@
 from ._pywt import dwt, idwt, swt, downcoef, upcoef, dwt_axis, idwt_axis
 
 
-def dwt2(data, wavelet, mode='sym'):
+def dwt2(data, wavelet, mode='sym', axes=(-2, -1)):
```

Comment:
I think this is potentially confusing. I had expected ``axis=None``, which means over all axes, and that this function would only expect 2-D input. Why accept n-D input here when there's ``dwtn`` for that?